### DISCUSSION TITLE
Multi-arch manifest combination

### DISCUSSION THREAD

**Discussion by olnor18** [dotnet/dotnet-docker#2649] [Ideas]
Is it by technical limitation, choice or something else, that the different architectures are build as separate tags instead of combined as one? This practice is used by node and python for instance, and allows for a easier usage across architectures.  This could either be achieved at build-time with buildx or after-the-fact with something like manifest-tool.

**Comment by mthalman** [comment]
There are multi-arch tags.  For example, `5.0` and `5.0-alpine` are examples of multi-arch tags.  You can find more information on this in the [supported tags](https://github.com/dotnet/dotnet-docker/blob/main/documentation/supported-tags.md#shared-tags) documentation.

**Reply by olnor18** [reply]
~~Nice! Are there any plans there any plans on including this on 3.1 or are they left as is?~~
Edit: Read the docs better. Saw that they already have multiarch for 3.1. I had specified 3.1-buster-slim, which wasn't multiarch

**Reply by mthalman** [reply]
It applies to all supported versions, including 2.1 and 3.1.  You can see the list of tags at the [Docker Hub pages](https://hub.docker.com/_/microsoft-dotnet) (e.g. [ASP.NET Core](https://hub.docker.com/_/microsoft-dotnet-aspnet))
